src/proc/split-filter.h

```diff
@@ -0,0 +1,30 @@
+/* License: Apache 2.0. See LICENSE file in root directory.
+Copyright(c) 2020 Intel Corporation. All Rights Reserved. */
+
+
+#pragma once
+
+#include "synthetic-stream.h"
+#include "option.h"
+
+namespace librealsense
+{
+    class split_filter : public generic_processing_block
+    {
+    public:
+        split_filter();
+
+    protected:
+        bool should_process(const rs2::frame& frame) override;
```

Comment:
Can be const

Reply:
It is not const in the base class - do you mean to change all the "should_process" methods, for all the filters?